Plugins/Json/include/Acts/Plugins/Json/DetrayJsonHelper.hpp

```diff
@@ -90,5 +93,42 @@ inline static void addVolumeLink(nlohmann::json& jSurface, int vLink) {
   jSurface["volume_link"] = vLink;
 }
 
+/// Determine the acceleration link from a grid
+///
+///
+///   brute_force = 0u,      // try all
+///   cartesian2_grid = 1u,  // rectangle, trapezoid, (triangle) grids
+///   cuboid3_grid = 2u,     // cuboid grid
+///   polar2_grid = 3u,      // ring/disc, annulus grids
+///   cylinder2_grid = 4u,   // 2D cylinder grid
+///   cylinder3_grid = 5u,   // 3D cylinder grid
+///
+/// @param casts are the grid axes cast types
+///
+/// @return the acceleration link idnetifier
+template <typename binning_values_t>
+inline static size_t accelerationLink(const binning_values_t& casts) {
+  // Default is `brute_force`
+  size_t accLink = 0u;
+  if (casts.size() == 2u) {
+    if (casts[0u] == binX && casts[1u] == binY) {
+      accLink = 1u;
+    } else if (casts[0u] == binR && casts[1u] == binPhi) {
+      accLink = 3u;
+    } else if (casts[0u] == binZ && casts[1u] == binPhi) {
+      accLink = 4u;
+    } else if (casts[0u] == binZ && casts[1u] == binR) {
```

Comment:
Is this possible in ACTS, to have 3D grid with only two axes?

Reply:
No, I tried to adapt to the detray grids, You do expect a 3D one, right?

Reply:
For the volume grid at some point, yes. The surface and material grids are all 2D (so far)

Reply:
Ok, I will just take a 2D grid and make a 3D grid out of it.